plugins/windows_dhcp.yaml

```diff
@@ -0,0 +1,41 @@
+# Plugin Info
+version: 0.0.1
+title: Windows DHCP
+description: Log parser for Windows DHCP
+parameters:
+  file_path:
+    label: Log Path
+    description: The absolute path to the Microsoft IIS logs. Simple globbing can be used to collect multiple log files
+    type: string
+    default: "C:/Windows/System32/dhcp/DhcpSrvLog-*.log"
+  start_at:
+    label: Start At
+    description: Start reading file from 'beginning' or 'end'
+    type: enum
+    valid_values:
+      - beginning
+      - end
+    default: end
+
+# Set Defaults
+{{$file_path := default "C:/Windows/System32/dhcp/DhcpSrvLog-*.log" .file_path}}
```

Comment:
Have wee seen this work with the forward slashes? I don't remember how our config behaves with windows paths

Reply:
I can confirm that forward slashes do work with the agent on windows.